README.md

```diff
@@ -12,7 +12,7 @@ This is an [Outreachy](https://www.outreachy.org/) Internship project.
 
 ## Issue Tracker
 
-This project uses [Phabricator](https://phabricator.wikimedia.org/project/board/6283/) to track issues and we would advice against using Github issue traking for bugs
+This project uses [Phabricator](https://phabricator.wikimedia.org/project/board/6283/) to track issues and we would advise against using Github issue tracking for bugs
```

Comment:
nit: the GitHub issues feature has been disabled on this repo as well as on toolhunt-ui since this was first written, so using it isn't an option anymore.

Reply:
Removed.  ;)